mkt/developers/tasks.py

```diff
@@ -121,7 +122,7 @@ def run_validator(file_path, url=None):
     # uploaded file is on the local filesystem.
     temp_path = tempfile.mktemp()
     with open(temp_path, 'wb') as local_f:
-        with storage.open(file_path) as remote_f:
+        with private_storage.open(file_path) as remote_f:
```

Comment:
(I wondered for a second if we needed to explicitly copy paths from one storage to another, but I guess we don't since signing will take care of that ? Unless something specific needs to be done for hosted apps, but I don't think so)